server/views/pages/find-unallocated-cases.njk

```diff
@@ -56,7 +56,43 @@
 } %}
 
 {% for item in unallocatedCases %}
-  {% if item.outOfAreaTransfer == true %}
+  {% if item.apopExcluded == true %}
+
+    {%- set tableRow = [
+            {
+                text: tripleCell(
+                  '<strong>' + item.name + '</strong>',
+                  item.crn,
+                  'govuk-body-s govuk-!-display-inline-block',
+                  '<span class="govuk-tag--orange">Restricted access</span>'
+                ),
+                attributes: { "data-sort-value": item.crn }
+            },
+            { text: 'This is a restricted case. Check NDelius if you require access or further information.', attributes: { "data-sort-value": item.tierOrder }, colspan: 5 }
+        ] -%}
+  {% elif item.excluded == true %}
+    {%- set tableRow = [
+            {
+                text: tripleCell(
+                  '<a href="/pdu/' + pduCode +'/' + item.crn + '/convictions/' + item.convictionNumber +'/case-view" class="govuk-link--no-visited-state" aria-label="Review case" data-qa-link="' + item.crn + '-' + item.convictionNumber +'"">' +  item.name + '</a>',
+                  item.crn, 
+                  'govuk-body-s govuk-!-display-inline-block',
+                  '<span class="govuk-tag--orange">Restricted access</span>'
+                ),
+                attributes: { "data-sort-value": item.crn }
+            },
+            { text: item.tier, attributes: { "data-sort-value": item.tierOrder } },
+            { text: (item.sentenceDate | dateFormat), attributes: {
+                "data-sort-value": item.sentenceDate
+            } },
+            { text: item.handoverDate, attributes: { "data-sort-value": item.handoverDate } },
+            {
+                text: doubleCell(item.primaryInitialAppointment,item.secondaryInitialAppointment, 'maw-secondary' ),
+                attributes: { "data-sort-value": item.primaryInitialAppointment }
+            },
+            { text: doubleCell(item.primaryStatus, item.secondaryStatus, 'govuk-body-s')}
+        ] -%}
+  {% elif item.outOfAreaTransfer == true %}
```

Comment:
I assume an item can be excluded ( _**{% elif item.excluded == true %}**_) **AND** outOfAreaTransfer.
Would we need the out of area message here too?